custom-connectors/Staffbase/apiProperties.json

```diff
@@ -0,0 +1,43 @@
+{
+  "properties": {
+    "connectionParameters": {
+      "api_key": {
+        "type": "securestring",
+        "uiDefinition": {
+          "displayName": "API-Schl\u00fcssel",
+          "description": "\"API-Schl\u00fcssel\" f\u00fcr diese API",
+          "tooltip": "Stellen Sie \"API-Schl\u00fcssel\" bereit.",
+          "constraints": {
+            "tabIndex": 2,
+            "clearText": false,
+            "required": "true"
+          }
+        }
+      },
+      "hostingURL": {
+        "type": "string",
+        "uiDefinition": {
+          "constraints": {
+            "required": "true"
+          },
+          "description": "Specify your hosting url: hostingURL",
+          "displayName": "Hosting URL",
+          "tooltip": "Provide hosting URL here"
+        }
+      }
+    },
+    "iconBrandColor": "#007ee5",
```

Comment:
@Nm-jens Please make sure that the given iconBrandColor is the background color of your connector's icon.

Documentation - [Icon](https://docs.microsoft.com/en-us/connectors/custom-connectors/certification-submission#icon)